CIP-1694/README.md

```diff
@@ -0,0 +1,664 @@
+* Relative to a percentage `P`, a DRep (SPO) **vote threshold has been met** if the sum of the relative stake controlled by the DReps (SPOs) voting 'yes' to a governance action minus the sum of the relative stake controlled by the DReps (SPOs) voting 'no' is at least `P`.
+
+> **Note**
+> There are several alternative definitions for "the total stake in circulation":
+>  1. The sum of the UTxO and the rewards accounts.
+>  2. The sum of the UTxO, the rewards accounts, the fee pot, and the deposit pot.
+>  3. The total ADA supply (i.e. 45 billion ADA) minus the reserves[^3].
+>  We leave the choice open for discussion.
+
+#### New Protocol Parameters
+
+New protocol parameters will be needed for the following:
+
+* the governance action deposit amount
+* governance action expiration (as a number of epochs from the current epoch)
+* thresholds for treasury withdrawals
+* each of the ratification thresholds
+
+As described above, some  of these will be updatable; others will be hard coded.
+
+// TODO: Decide on the initial parameter values and whether they should be updatable.
+
+// TODO: Decide on coherence conditions on the voting thresholds. For example, the threshold for a motion of no-confidence should arguably be higher than that of a minor treasury withdrawal.
+
+In addition, the initial Constitutional Committee and the initial Constitution will need to be defined as part of a bootstrap process.
+
+#### Changes to the existing ledger rules
+
+* The `PPUP` transition rule will be rewritten and moved out of the `UTxO` rule and into the `LEDGER` rule as a new `TALLY` rule.
+
+  It will process the governance actions and the votes, ratify them, and stage governance actions for enactment in the current or next epoch, as appropriate.
+
+* The `NEWEPOCH` transition rule will be modified.
+* The `MIR` sub-rule will be removed.
+* A new `ENACTMENT` rule will be called immediately after the `EPOCH` rule. This rule will enact governance actions that have previously been ratified.
+* The `EPOCH` rule will no longer call the `NEWPP` sub-rule or compute whether the quorum is met on the PPUP state.
+
+#### More on DRep incentives
+
+The DReps arguably need to be compensated for their work.
+
+The corresponding incentive mechanisms need to be specified, with the funds probably coming from the per-epoch treasury allocation.
+Performance constraints will also need to be considered since it would be problematic if millions of DReps were expected to vote on each governance action. Some incentive options to ensure a manageable number of DReps include:
+
+* Requiring a large deposit when registering oneself as a DRep.
+* An incentive scheme similar to the stake pool reward scheme is used to limit individual rewards. (Note that this is probably not enough on its own, as many voters may wish to be their own DRep regardless of payment).
+* Some form of a per-epoch randomized sub-committee of DReps which takes stake weight into account.
+
```

Comment:
Holding the position of DRep shouldn't be all perks, but should also come with the responsibility to effectively represent the interests of the ADA holders who have delegated their voting power to the DRep.
The failure to cast a vote over a specified number of consecutive epochs, should result in an automatic retirement of the DRep, or even fines to be paid to the treasury.